### PR TITLE
docs: Add `CONTRIBUTING` and `README`

### LINKED ISSUE #54: Create a contribute guide

### PR DESCRIPTION
**Pull request by eureka-cpu**
Closes #54 

This adds some small updates to the SwaySwap `readme` as well as the `contributing` guide.

### CODE REVIEW

**Comment by eureka-cpu** [comment]
@luizstacio did we get rid of the `client/readme` in a previous PR, it's showing that I've added it but just want to make sure we don't need it

**Review by pedronauck — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Contributing To SwaySwap
+
+Thanks for your interest in contributing to SwaySwap! This document outlines the process for installing dependencies and setting up SwaySwap for development, as well as some conventions on contributing to SwaySwap.
+
+If you run into any difficulties getting started, you can always ask questions on our [Discord](https://discord.gg/xfpK4Pe).
+
+## Setting Up a Development Workspace
```
[the remark this answers is not in the document]
Comment (eureka-cpu) [reply CONTRIBUTING.md:7]: Which part, the discord link, or `Setting Up a Development Workspace`?

**Comment by eureka-cpu** [comment]
> @luizstacio @adlerjohn The setup information that is in the `README`s is sort of scattered. Do we really want a bunch of `README` docs that all point to one another? I would suggest moving the setup information into `CONTRIBUTING` to consolidate so that anyone wanting to contribute won't have to jump around from doc to doc just to get started. Same goes for `packages/app/README` since it shows how to view the deployment.

I've gone ahead and updated this PR to reflect how it could be more clear for contributors

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+- `VITE_CONTRACT_ID` - Id (address) of the deployed swayswap contract
+
+- `VITE_TOKEN_ID=` - Id (address) of the deployed token contract
+
+- `VITE_RECAPTCHA_SITE_KEY` - The site key is used to invoke recaptcha service on the website
+
+- `VITE_FUEL_FAUCET_URL=` - Link for the fuel faucet
+
+After deploying the SwaySwap and token contracts make sure to copy and paste the SwaySwap and token contract IDs into `.env` before running the development server.
```
Comment (matt-user) [inline CONTRIBUTING.md:77]: Maybe change this text to "Before starting the development server make sure the contract id and token id environment variables are set to the corresponding contract addresses." from the app/readme as I believe its slightly more accurate.

**Review by matt-user — DISMISSED** [review]
Looks good.  I just have one nit comment.

File: CONTRIBUTING.md (hunk 2 above)
Comment (eureka-cpu) [reply CONTRIBUTING.md:77]: Will do thx!

**Review by matt-user — APPROVED** [review]

**Review by luizstacio — APPROVED** [review]
⚡️⚡️⚡️⚡️⚡️❤️❤️❤️❤️❤️

**Pull request merged**